src/PathOrderOptimizer.h

```diff
@@ -501,7 +501,19 @@ class PathOrderOptimizer
                     score += 1000; //1 meter penalty.
                 }
             }
-            if(score < best_score)
+
+            constexpr float EPSILON = 25.0;
+            if(best_score - EPSILON <= score && score <= best_score + EPSILON)
+            {
+                // add breaker for two candidate starting location with similar score
+                // if we don't do this then we (can) get an un-even seam
+                Point tie_breaker_point = target_pos != Point(0, 0)
+                    ? Point(0, 0)       // break ties by picking the point that is closest to the origin
+                    : Point(0, 100000); // unless the target position is the origin, then pick some other location
+                best_point = vSize2(best_point - tie_breaker_point) < vSize2(best_point - tie_breaker_point) ? best_point : here;
+                best_score = std::min(best_score, score);
```

Comment:
I think you could remove the above code and just leave this in as the tie breaker, that seems to be how it functions now.

Reply:
Well there can still be a small difference (at most epsilon) between the two numbers, I'd prefer to keep it.